worlds/oot_soh/location_access/dungeons/bottom_of_the_well.py

```diff
@@ -1,16 +1,217 @@
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_POT10.value, lambda state: can_break_pots(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_POT11.value, lambda state: can_break_pots(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_POT12.value, lambda state: can_break_pots(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_MQ_BASEMENT_SUNS_SONG_FAIRY.value, lambda state: can_use(Items.SUNS_SONG.value, state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_GRASS1.value, lambda state: can_cut_shrubs(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_GRASS2.value, lambda state: can_cut_shrubs(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_GRASS3.value, lambda state: can_cut_shrubs(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS1.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS2.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS3.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS4.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS5.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS6.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS7.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS8.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)],
+        [Locations.BOTTOM_OF_THE_WELL_BASEMENT_BEHIND_ROCKS_GRASS9.value, lambda state: can_cut_shrubs(state, world) and blast_or_smash(state, world)]
+    ])
+    # Connections
+    connect_regions(Regions.BOTTOM_OF_THE_WELL_BASEMENT.value, world, [
+        [Regions.BOTTOM_OF_THE_WELL_SOUTHWEST_ROOM.value, lambda state: is_child(state, world) and can_pass_enemy(Enemies.BIG_SKULLTULA.value, state, world)],
+        [Regions.BOTTOM_OF_THE_WELL_BASEMENT_USEFUL_BOMB_FLOWERS.value, lambda state: blast_or_smash(state, world) or can_use(Items.DINS_FIRE.value, state, world) or (can_use(Items.STICKS.value) and can_do_trick("RT BOTW Basement", state, world))] # NOT 100% SURE HOW TO TRANSLATE HERE. Has to do with Age: Entrance(RR_BOTTOM_OF_THE_WELL_BASEMENT_USEFUL_BOMB_FLOWERS, []{return Here(RR_BOTTOM_OF_THE_WELL_BASEMENT, []{return logic->BlastOrSmash() || logic->CanUse(RG_DINS_FIRE) || (logic->CanUse(RG_STICKS) && ctx->GetTrickOption(RT_BOTW_BASEMENT));});}),
+    ])
```

Comment:
I think this is corrrect as you have it. The Here is not relevant for us as it's how Ship deals with a combination of age difference + entrance rando + time of day, which isn't relevant for us.